src/broadcast/defs_broadcast.h

```diff
@@ -23,8 +23,10 @@
 #define BROADCAST_BITRATE_48KBPS 48
 #define BROADCAST_BITRATE_32KBPS 32
 
+// WARNING! def_recordings and defs_broadcast format codes have to be the same
```

Comment:
Instead of adding a warning: How about defining them only once thereby fixing the root cause?

Reply:
done